observability/power_monitoring/power-monitoring-release-notes.adoc

```diff
@@ -15,6 +15,40 @@ These release notes track the development of {PM-title} in the {product-title}.
 
 For an overview of the {PM-operator}, see xref:../power_monitoring/power-monitoring-overview.adoc#power-monitoring-about-power-monitoring_power-monitoring-overview[About {PM-shortname}].
 
+[id="power-monitoring-release-notes-0-3"]
+== {PM-shortname-c} 0.3 (Technology Preview)
+
+This release includes the following version updates:
+
+* {PM-kepler} 0.7.12
+* {PM-operator} 0.15.0
+
+The following advisory is available for {PM-shortname} 0.3:
+
+* link:https://access.redhat.com/errata/RHEA-2024:9961[RHEA-2024:9961]
+
+[id="power-monitoring-known-issue-0-3-features"]
+=== Known Issue
+
+* Currently, {PM-kepler} may report incorrect or unknown CPU architecture when deployed on BareMetal servers and Virtual Machines.
```

Comment:
@shreyasiddhartha Do we add link to bug id for tracking in release notes?

Reply:
@vprashar2929 Tried doing that but my PR failed saying "Couldn't include links that requires login". So I am not adding links for now.